Create lets-roll.mdx (#138)
rollkit/docs: blog/let's-roll.mdx

---
slug: Let's Roll- Rollkit's community call
title: "Let's Roll- Rollkit's community call"
authors: [AlCJoseph]

> tags: [Rollkit]

Reviewer: ```suggestion
tags: [Rollkit][Community]
```

Author: tag here is not well written, it should be [Rollkit, Community]

> description: Rollkit is kicking off its community call, Let's Roll, to to bring together community members to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.

Reviewer: ```suggestion
description: Rollkit is kicking off its community call, Let's Roll, to bring together community members to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.
```

---

![Rollkit blog cover](../static/img/lets-roll/lets-roll.jpg)

Last Sunday, we released the first [research implementation](https://rollkit.dev/blog/sovereign-rollups-on-bitcoin/) that allows Rollkit rollups to use Bitcoin for data availability. 

> [This announcement](https://twitter.com/RollkitDev/status/1632438374513676288) has sparked a lot of curiosity and excitement in the community. Many developers are starting to imagine what the future of sovereign rollups could look like on Bitcoin and other data availability layers. They’re looking for ways to innovate using this integration, to replicate it on other layer 1 blockchains, to identify challenges and to suggest improvements.
>  
> We believe that there’s a lot of innovation to be done and a lot of challenges to be solved, and that together as a community, we can collaborate on building the future of rollup frameworks.
> 
> That’s why we’re happy to announce that we’re launching a Rollkit’s community call: Let’s Roll.

Reviewer: ```suggestion
[This announcement](https://twitter.com/RollkitDev/status/1632438374513676288) has sparked a lot of curiosity and excitement in the community. Many developers are starting to imagine what the future of sovereign rollups could look like on Bitcoin and other data availability layers. They’re looking for ways to innovate using this integration, replicate it on other layer-1s, identify challenges, and suggest improvements.
 
We believe that there’s a lot of innovation to be done and many challenges to be solved and that we can collaborate as a community to build the future of rollup frameworks.

We’re happy to announce that we’re launching a Rollkit’s community call: Let’s Roll.
```

> ## TL;DR:

Reviewer 2: no need `:` at the end

Author: true

-Rollkit is launching its community call, Let’s Roll. You can subscribe to the community calendar [here](https://calendar.google.com/calendar/u/0/embed?src=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com&ctz=America/Toronto)

> -Builders using Rollkit can now present their projects at Let’s Roll. Reach out to us on [Telegram](https://t.me/rollkit) 

Reviewer 2: should the telegram link be join link instead of browse the telegram group?

Author: it could be but don't both links gets you to the same page on a laptop? only on phones it's different right?
here's the join link https://t.me/+THiNuoYyUEFjZjcx

## Just merge it

> Rollup development is rapidly evolving, with new types of rollups emerging, like sovereign and pessimistic rollups, and with new possibilities of using layer 1 blockchains as data availability layers. Rollkit is at the forefront of the rollup movement offering an open source, public good and modular framework for rollups. 

Reviewer 3: ```suggestion
Rollup development is rapidly evolving, with new types of rollups such as sovereign and pessimistic rollups emerging, and with new possibilities of using layer 1-blockchains as data availability layers. Rollkit is at the forefront of the rollup movement offering a modular framework for rollups as an open-source public good. 
```

> Rollup development is rapidly evolving, with new types of rollups emerging, like sovereign and pessimistic rollups, and with new possibilities of using layer 1 blockchains as data availability layers. Rollkit is at the forefront of the rollup movement offering an open source, public good and modular framework for rollups. 
> 
> However, Rollkit's true potential lies in its community of developers and users who bring their unique perspectives, skills, and expertise to the table. When Rollkit integrates a new data availability layer, its community members are best suited to improve this integration and use it in ways that truly benefit the community. Therefore, we invite developers from different communities and skill sets to join us and contribute to Rollkit’s development.
> 
> By contributing to Rollkit, we reduce fragmentation, duplicated features and bugs, and we ensure the continuous improvement of Rollkit. By doing that we forge a stronger and bigger community. 
> 
> One framework is bigger than the sum of all its forks. Together we can build something for all developers and builders to easily use.
> 
> Our promise, as core contributors, is to do our best to make “just merge it” a reality. We strive to empower developers to contribute to Rollkit and ensure that their contributions are integrated. 
> 
> That's why we believe it's time to bring our community together to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.

Reviewer: ```suggestion
Rollup development is rapidly evolving, with new types of rollups emerging, like sovereign and pessimistic rollups, and with new possibilities of using layer 1 blockchains as data availability layers. Rollkit is at the forefront of the rollup movement, offering an open-source, public good and modular framework for rollups. 

However, Rollkit's true potential lies in its community of developers and users who bring unique perspectives, skills, and expertise. For example, when Rollkit integrates a new data availability layer, its community members are best suited to improve this integration and use it in ways that truly benefit the community. Therefore, we invite developers from different communities and skill sets to join us and contribute to Rollkit’s development.

By contributing to Rollkit, we reduce fragmentation, duplicated features and bugs, and we ensure the continuous improvement of Rollkit. By doing that, we forge a more robust and bigger community. 

One framework is more significant than the sum of all its forks. Together we can build something for all developers and builders to use easily.

Our promise, as core contributors, is to do our best to make “just merge it” a reality. We strive to empower developers to contribute to Rollkit and ensure their contributions are integrated. 

That's why we believe it's time to unite our community to share knowledge, collaborate on new ideas, and inspire each other to push the limits of what's possible with Rollkit.
```

## What is Let’s Roll?

> Let's Roll is a dynamic community-focused meeting that brings together all Rollkit’s contributors to drive the innovation and development of Rollkit.
> 
> This meeting serves as a platform for sharing knowledge, discussing the latest updates, and exploring new ideas. Rollkit's contributors will demonstrate the most recent features and advancements, while community members will have the chance to ask questions, provide feedback, and collaborate on new projects.
> 
> Building open source means encouraging collaboration and teamwork among all developers. Sharing code and ideas will help us build a better framework more efficiently.
> 
> The meeting will be hosted on [Zoom](https://us06web.zoom.us/j/83308619122). Subscribe to the following [Google calendar](https://calendar.google.com/calendar/u/0/r?cid=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com) so that you can be notified of future community calls.

Reviewer: ```suggestion
Let's Roll is a dynamic community-focused meeting that brings together all of Rollkit’s contributors to drive the innovation and development of Rollkit.

This meeting is a platform for sharing knowledge, discussing the latest updates, and exploring new ideas. Rollkit's contributors will demonstrate the most recent features and advancements, while community members can ask questions, provide feedback, and collaborate on new projects.

Building open source means encouraging collaboration and teamwork among all developers. Sharing code and ideas will help us build a better framework more efficiently.

The meeting will be hosted on [Zoom](https://us06web.zoom.us/j/83308619122). In addition, subscribe to the following [Google calendar](https://calendar.google.com/calendar/u/0/r?cid=c_11a8d6a470ca9c07b131ab1ea213ac124a3061a9d6986bb208d88afbcc4bb3d2@group.calendar.google.com) so that you can be notified of future community calls.
```

## Conclusion

> Let's Roll is an exciting opportunity for the Rollkit community to come together, share ideas, and collaborate on the future of this powerful framework.
> With demos of the latest developments, discussion of the roadmap and vision, and a dedicated Q&A and feedback session, Let's Roll is an opportunity to stay informed and stay engaged with the project.
> 
> We invite everyone who is interested in Rollkit and rollup development to join us. Your feedback, ideas, and input are essential to the success of Rollkit, and we look forward to seeing what we can achieve together.
> 
> So, join us, roll with us, and let's shape the future of rollup frameworks together.

Reviewer: ```suggestion
Let's Roll is an exciting opportunity for the Rollkit community to come together, share ideas, and collaborate on the future of this powerful framework.
With demos of the latest developments, a discussion of the roadmap and vision, and a dedicated Q&A and feedback session, Let's Roll is an opportunity to stay informed and engaged with the project.

We invite everyone interested in Rollkit and rollup development to join us. Your feedback, ideas, and input are essential to the success of Rollkit, and we look forward to seeing what we can achieve together.

So, join us, roll with us, and let's shape the future of rollup frameworks together.
```
